outlookmail/Module.md

```diff
@@ -1,29 +1,28 @@
 ## Overview
-Ballerina connector for Microsoft Outlook mail provides access to Microsoft Outlook mail service in Microsoft Graph v1.0 via Ballerina language easily. It provides capability to perform more useful functionalities provided in Microsoft outlook mail such as sending messages, listing messages, creating drafts, mail folders, deleting and updating messages etc. 
+Microsoft Outlook mail API provides capability to access Outlook mail operations related to messages, attachments, drafts, 
```

Comment:
I like the earlier description :)

Reply:
Shall we rephrase this to: Ballerina connector for Microsoft Outlook mail provides access to the Microsoft Outlook mail service in Microsoft Graph v1.0 via the [Ballerina language](https://ballerina.io/). It provides the capability to perform more useful functionalities provided in Microsoft outlook mail such as sending messages, listing messages, creating drafts, mail folders, deleting messages, updating messages, etc.